cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -464,6 +528,18 @@ def _measurement(two_qubit_circuit: circuits.Circuit) -> np.ndarray:
     return TomographyResult(rho)
 
 
+def _create_parallel_rb_circuit(
+    qubits: Iterator['cirq.GridQubit'], num_cfds: int, c1: list, cfd_mats: np.ndarray
+) -> 'cirq.Circuit':
+    circuits_to_zip = [_random_single_q_clifford(qubit, num_cfds, c1, cfd_mats) for qubit in qubits]
+    circuit = circuits.Circuit.zip(*circuits_to_zip)
+    measure_moment = circuits.Moment(
+        ops.measure_each(*qubits, key_func=lambda q: f"q{q.row}_{q.col}")  # type: ignore
+    )
+    circuit_with_meas = circuits.Circuit.from_moments(*(list(circuit.moments) + [measure_moment]))
```

Comment:
Can use more than one argument instead of creating a list first:
```suggestion
    circuit_with_meas = circuits.Circuit.from_moments(*circuit.moments, measure_moment))
```

Reply:
Good catch, thanks!